sdk/cosmos/azure-cosmos/docs/StatusCodes.md

```diff
@@ -0,0 +1,53 @@
+
+
+
+## Status codes
+
+| Status code | Substatus code | Always transient | Additional info |
+| -----------------:|-----------------------:|:-----------------------:|:----------------------|
+|200|0|| `OK`|
+|201|0|| `Created` - returned for createItem or upsertItem when a new document was created|
+|204|0|| `No Content` - returned when no payload would ever be returned - like for delete operations|
+|304|0|| `Not Modified` - will be returned for `ReadFeed` and `ChangeFeed` operations to indicate that there are no more changes|
+|400|\*|| `Bad Request` - indicates that the client violated some protocol constraint. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-bad-request) for more details.|
+|400|1001|| `Bad Request/Partition key mismatch` - indicates that the PartitionKey defined in the point operation does not match the partition key value being extracted in the service form the document's payload based on the `PartitionKeyDefinition` of the container. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-bad-request) for more details.|
+|401|0||`Unauthorized` - indicates that teh client used invalid credentials. This could mean an invalid key when using `MasterKey`-based authentication, it could mean there is a time-synchronization issue or when using AAD that the AAD credentials are not correctly set-up. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-unauthorized) for more details.|
+|403|\*||`Forbidden` - indicates that the service rejected the request due to missing permissions. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-forbidden)|
+|403|3|x|`Forbidden/WriteForbidden` - indicates that the client attempted a write operation against a read-only region in a single write region set-up.|
+|403|1008|x|`Forbidden/AccountNotFound` - indicates that the client attempted a read or write operation against a replica that did not have information about the database account.|
+|403|5300||`Forbidden/AADForMetadata` - indicates that the client attempted a metadata operation (like creating, deleting or modifying a container/database) when using AAD authentication. This is not possible via the Data plane SDK. To execute control plane operatiosn with AAD authentication, please use the management SDK. See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-forbidden#partition-key-exceeding-storage) and also the [Service quota docuemntation](https://learn.microsoft.com/azure/cosmos-db/concepts-limits#provisioned-throughput) for more details|
+|403|1014||`Forbidden/LogicalPartitionExceedsStorage` - indicates that the data size of a logical partition exceeds the service quota (currently 20 GB). See [TSG](https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-forbidden#non-data-operations-are-not-allowed) for more details|
```

Comment:
Included in https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-forbidden already

Reply:
Already referenced